src/App/components/FormComponents/Radio.js

```diff
@@ -24,7 +24,7 @@ const Radio = ({ id, checked, disabled, label, group, groupTitle, options, optio
                             <input {...attrs} id={id} defaultChecked={checked}/>{"\n"}
                             <label htmlFor={id}>{label}</label>{"\n"}
                         </div>)}
-                        {errorMessage && <><div className="help-block">{errorMessage}</div>{"\n"}</>}
+                        {errorMessage && <><div className="help-block"><i className="material-icons">error</i>{errorMessage}</div>{"\n"}</>}
```

Comment:
Triangle icon in figma
![image](https://user-images.githubusercontent.com/44967773/174799724-35d44731-244d-4a47-81b0-04617a3b8b23.png)